### PR TITLE
updated docker commands in tox

### PR DESCRIPTION
**Pull request by nsahai**
## Description
Added docker rmi to remove only local images and not  from a registry

### CODE REVIEW

File: tests/aiohttp/tox.ini [hunk 1]
```diff
@@ -35,5 +35,6 @@ commands =
     docker-compose up -d flask_app_c
     pytest -rPx test_aiohttp_4.py
     docker-compose stop
+    docker-compose down --rmi all
```
Comment (jcchavezs) [inline tests/aiohttp/tox.ini:38]: Do we still need the previous step?

File: tests/docker/tox.ini [hunk 2]
```diff
@@ -32,8 +32,9 @@ commands =
     docker-compose rm -v -f
     docker-compose up -d flask_app
     docker-compose stop flask_app
-    docker ps -aq
-    docker image prune -a -f
-    docker volume prune -f
+   # docker ps -aq
+   # docker image prune -a -f
+   # docker volume prune -f
+    docker-compose down --rmi all
```
Comment (jcchavezs) [inline tests/docker/tox.ini:38]: shall we do -v too?

File: tests/docker/tox.ini [hunk 3]
```diff
@@ -32,8 +32,9 @@ commands =
     docker-compose rm -v -f
```
Comment (jcchavezs) [inline tests/docker/tox.ini:32]: I think we don't need this step or should be executed instead of two steps above.

File: tests/docker/tox.ini [hunk 4]
```diff
@@ -32,8 +32,9 @@ commands =
...
     docker-compose up -d flask_app
     docker-compose stop flask_app
```
Comment (jcchavezs) [inline tests/docker/tox.ini:34]: I don't seem to get this code. We up a container and then immediately stop it, what is the point? Is that we expect the test to run in between? That could lead to race conditions.

File: tests/gunicorn/tox.ini [hunk 5]
```diff
@@ -20,11 +20,11 @@ whitelist_externals =
 setenv =
   PYTHONPATH=../../src
   ENABLE_INSTRUMENTATION=False
-  HT_REPORTING_TRACE_REPORTER_TYPE=zipkin
+  HT_TRACES_EXPORTER=zipkin
```
Comment (jcchavezs) [inline tests/gunicorn/tox.ini:23]: We changed this env var to HT_REPORTING_TRACE_REPORTER_TYPE

File: tests/gunicorn/tox.ini [hunk 6]
```diff
@@ -20,11 +20,11 @@ whitelist_externals =
...
   HT_REPORTING_ENDPOINT=http://localhost:9411/api/v2/spans
-  HT_LOG_LEVEL={env:HT_LOG_LEVEL:}
+  HT_LOG_LEVEL=DEBUG
```
Comment (jcchavezs) [inline tests/gunicorn/tox.ini:25]: Curious about the reason for this change. This was introduced to let the user decide about the log level when running the tests.

File: tests/mysql/tox.ini [hunk 7]
```diff
@@ -1,6 +1,6 @@
 [tox]
+envlist = py37, py38, py39
```
Comment (jcchavezs) [inline tests/mysql/tox.ini:2]: Any particular reason for this change?

File: tests/aiohttp/tox.ini (hunk 1 above)
Comment (nsahai) [reply tests/aiohttp/tox.ini:38]: I think that can be remove  but let me run some local test to validate.

File: tests/docker/tox.ini (hunk 4 above)
Comment (nsahai) [reply tests/docker/tox.ini:34]: agreed! will  test and validate

File: tests/docker/tox.ini (hunk 3 above)
Comment (nsahai) [reply tests/docker/tox.ini:32]: We might need for MySQL and postgres as we are creating volumes in that but I'll check.

File: tests/gunicorn/tox.ini (hunk 5 above)
Comment (nsahai) [reply tests/gunicorn/tox.ini]: Fixed

File: tests/gunicorn/tox.ini (hunk 6 above)
Comment (nsahai) [reply tests/gunicorn/tox.ini:25]: fixed

File: tests/docker/tox.ini (hunk 4 above)
Comment (nsahai) [reply tests/docker/tox.ini:34]: removed this test case

File: tests/docker/tox.ini (hunk 2 above)
Comment (nsahai) [reply tests/docker/tox.ini:38]: removed this test case

File: tests/gunicorn/tox.ini [hunk 8]
```diff
@@ -42,4 +42,5 @@ commands =
     docker-compose up -d flask_app3
     pytest -rPx test_gunicorn_3.py
     docker-compose stop
```
Comment (jcchavezs) [inline tests/gunicorn/tox.ini:44]:
```suggestion
```

**Review by jcchavezs — APPROVED** [review]

**Pull request merged**